install/bunkerweb-install.sh

```diff
@@ -23,19 +23,19 @@ $STD apt-get install -y lsb-release
 $STD apt-get install -y debian-archive-keyring
 msg_ok "Installed Dependencies"
 
-msg_info "Installing Nginx v1.20.0"
-wget -qO- https://nginx.org/keys/nginx_signing.key | gpg --dearmor >/usr/share/keyrings/nginx-archive-keyring.gpg
-echo "deb [signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg] http://nginx.org/packages/debian `lsb_release -cs` nginx" >/etc/apt/sources.list.d/nginx.list
+msg_info "Installing Nginx v1.26.1"
+curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor | $STD tee /usr/share/keyrings/nginx-archive-keyring.gpg >/dev/null
```

Comment:
why would you change the commands?

Reply:
I mirrored our official documentation: https://docs.bunkerweb.io/latest/integrations/#__tabbed_1_1